unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -36,6 +36,7 @@ public class AvatarRenderer : MonoBehaviour, IAvatarRenderer
         internal FacialFeatureController mouthController;
         internal AvatarAnimatorLegacy animator;
         internal StickersController stickersController;
+        internal SkinnedMeshRenderer combinedMeshRenderer;
```

Comment:
This property only brings complexity (you can use (`avatarMeshCombiner.renderer`).

Also this whole implementation will probably get in the way of the GPUSkinning. When the `SkinnedMeshRenderer` is swapped for a `MeshRenderer`
Take a look at how has been implemented there: https://github.com/decentraland/unity-renderer/blob/3e038c38d6c25b0b43b819457ba60b45d7370867/unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

A property simply returns whatever renderer we are using and that property is used to set stuff like SSAO or Dither.

Reply:
excellent! I changed back the class modifications I made.